deployer/aws-lambda/content-origin-request/index.js

```diff
@@ -19,26 +19,41 @@ function slugToFolder(slug) {
     .join("/");
 }
 
+function redirect(location, { permanent = false } = {}) {
+  /*
+   * Create and return a redirect response.
+   */
+  return {
+    status: permanent ? "301" : "302",
+    statusDescription: permanent ? "Moved Permanently" : "Found",
+    headers: {
+      location: [
+        {
+          key: "Location",
+          value: location,
+        },
+      ],
+    },
```

Comment:
I think we should also have:
```javascript
      'cache-control': [{
            key: 'Cache-Control',
            value: `max-age=${cacheControlSeconds}`
          }],
```
and perhaps change line 22 to:

```javascript
function redirect(location, { permanent = false,  cacheControlSeconds = 3600 * 24 } = {}) {
```

Reply:
Perhaps, we could use the fact that if it's permanent, it can cache a LOT longer. 
E.g. 
```
if (permanent) {
  // If it's a permanent redirect, bla bla
  cacheControlSeconds *= 30;
}
```

Reply:
Note, it should probably be
```javascript
value: `max-age=${cacheControlSeconds},public`
```

Reply:
ACTUALLY, we chatted about this in our 1:1 and concluded that there are cases where we might want to NOT have an aggressive cache control. E.g. requests for `/docs/Web` should respect the `Accept-Language` header and thus it needs to be a private cache-control.

Basically, we need this function here to be flexible and unassuming but that we should, in places where we know we want it, be able to set cache control on the redirects (e.g. URLs that end with a `/`).